.github/workflows/mps-tests.yml

```diff
@@ -42,64 +42,29 @@ jobs:
     timeout-minutes: 60
 
     steps:
-      - name: Clean workspace
```

Comment:
@BanzaiTokyo I do not think we should do that. To have valid MPS support we should use the current runner. GHA m1 runner does not have MPS enabled in torch, we've already seen together.